src/mobile-token/MobileTokenContext.tsx

```diff
@@ -65,7 +65,10 @@ export const MobileTokenContextProvider: React.FC = ({children}) => {
   const {token_timeout_in_seconds} = useRemoteConfig();
   const hasEnabledMobileToken = useHasEnabledMobileToken();
 
-  const {enable_token_fallback: fallbackEnabled} = useRemoteConfig();
+  const {
+    enable_token_fallback: fallbackOnErrorEnabled,
```

Comment:
This remote config flag isn't very precisely named now, I would have called it `enable_token_fallback_on_error` if I created it now. But not changing it since (1) the effect of the flag shouldn't have changed at all and (2) we would need to maintain two fields in firebase for backwards compatibility.